docs/usage/self-hosted-configuration.md

```diff
@@ -292,6 +292,13 @@ Like this:
 
 ## dryRun
 
+You can choose from the following behaviors for the `dryRun` config option:
+
+- `null`: Performs a regular renovate run include creating/updating/deleting branches and PRs, this is the default value
+- `"extract"`: Performs extract only, prints results, exits
```

Comment:
I'm not sure what "extract only" means here, do we need to explain that in more detail here?